### PR TITLE
Added default config options for htmlhint

### PR DESCRIPTION
**Pull request by mrkoreye**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -60,7 +61,11 @@ export class Config {
     argvPrefix: 'config.',
     argvSeparator: '.',
     environment: process.env.NODE_ENV || 'development',
-    keypathSplitRegExp: /[\[\]\.]/
+    keypathSplitRegExp: /[\[\]\.]/,
+    htmlhint: {
+      "doctype-first": false,
+      "title-require": false
```
Comment (steve8708) [inline src/config.ts]: single quotes :)

**Comment by mrkoreye** [comment]
@steve8708 updated

File: src/config.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ export interface IConfigOptions {
   argvSeparator?: string;
   environment?: string;
   keypathSplitRegExp?: RegExp;
+  htmlhint?: Object;
```
Comment (steve8708) [inline src/config.ts]: this should be `any`

`Object` expects a completely generic object, aka `{}`

**Comment by mrkoreye** [comment]
@steve8708 updated

**Comment by steve8708** [comment]
:+1: :+1: :+1: :+1: :+1:

**Pull request merged**